source/manual/related-links-on-govuk.md

```diff
@@ -0,0 +1,72 @@
+
+Once these links have been created, a second job ingests the links via the Publishing API to be shown to users navigating the site.
+
+The code for the process of generating and ingesting related links can be found in the [govuk-related-links-recommender repository on GitHub](https://github.com/alphagov/govuk-related-links-recommender).
+
+## When we show the new related links
+
+Machine learning generated related links are created exclusively for Whitehall content, with a number of measures in place to ensure that we always show manually curated links set by publishers.
+
+- Related links are only generated for a number of content types, with exclusions in place for content that should not be linked _from_, linked _to_, or both. These exclusions can be found at https://github.com/alphagov/govuk-related-links-recommender/tree/master/src/config.
+- Generated related links are stored within the `suggested_ordered_related_items` property of a content item, while all existing and manually curated links continue to be stored within the `ordered_related_items` property.
+- If links exist with `ordered_related_items` for a content item, we do not show generated related links.
+- Requests need to have the header `Govuk-Use-Recommended-Related-Links` header set to `True` in order to show suggested related links - [this is set by the CDN for all requests](https://github.com/alphagov/govuk-cdn-config/blob/master/vcl_templates/www.vcl.erb#L238).
+
+
+## Concourse pipeline details
+
+Related links are generated and ingested via the following jobs, which are defined in the [govuk-related-links pipeline on Concourse](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-related-links).
+
+### `link-generation`
+
+This job runs the process to generate new related links for a subset of pages on GOV.UK, via the `provision-generation-machine` and `run_link_generation` scripts, which in turn triggers the Python script `run_all.py`.
+
+The related links generated from this process are stored in an S3 environment-specific `govuk-related-links` bucket.
+
+Currently, this job is scheduled to run once every three weeks via a `time` resource attached to the production `run-generation-production` step.
+
+### `link-ingestion`
+
+This job runs the process of ingesting related links via the Publishing API.
+
+Links are retrieved from the environment-specific S3 bucket mentioned previously and then fed (in batches) to the Publishing API. There is a 20 minute delay between batches.
+
+Currently, this job is triggered manually after reviewing a spreadsheet of the generated related links; it will be scheduled to run automatically in the near future.
```

Comment:
Can we nail down "in the near future" a bit more precisely?  Statements like that tend to hang around.

Reply:
One of our missions at the moment is to monitor related links, so by the end of September; I'll add that in.